int/hwsc-file-transaction-svc/proto/hwsc-file-transaction-svc.proto

```diff
@@ -2,11 +2,10 @@ syntax = "proto3";
 
 package hwscFileTransactionSvc;
 
-// Service: define the methods that gRPC server can expose to the client
 service FileTransactionService{
-    rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse){}
-    rpc UploadFile (stream Chunk) returns (FileTransactionResponse){}
-    rpc DownloadZipFiles (FileTransactionRequest) returns (stream Chunk){}
+    rpc get_status (FileTransactionRequest) returns (FileTransactionResponse){}
```

Comment:
Is there a reason why rpc's naming style changed to underscore_separated_names?

Proto naming convention says to name rpc's using CamelCase but maybe you guys had a reason?:
https://developers.google.com/protocol-buffers/docs/style#services

Reply:
The reason why the names of the rpc were switched from  `underscore style` to `CamelCase`  was because of [Pep-8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) standard. I might have recommended this due to 2nd nature.

Unfortunately, I was wrong recommending a style-change. This may look confusing and inconsistent in the `hwsc-app-gateway-svc`.

Please revert back to `CamelCase`